### DISCUSSION TITLE
please tell me how to  use react native firebase in ios giving error in pod install

### DISCUSSION THREAD

**Discussion by Lov9766** [invertase/react-native-firebase#6340] [Q&A]
pod install
RNFBApp: Using overridden static_framework value of 'true'
Adding a custom script phase for Pod RNFBApp: [RNFB] Core Configuration
RNFBAuth: Using overridden static_framework value of 'true'
Auto-linking React Native modules for target `project2`: RNFBApp and RNFBAuth
[Codegen] Generating ./build/generated/ios/React-Codegen.podspec.json
Analyzing dependencies
[Codegen] Found FBReactNativeSpec
RNFBApp: Using overridden static_framework value of 'true'
RNFBAuth: Using overridden static_framework value of 'true'
Downloading dependencies
Generating Pods project
/Users/lovkumar/project/project2/ios/Pods/Target Support Files/Pods-project2/Pods-project2.release.xcconfig
/Users/lovkumar/project/project2/ios/Pods/Target Support Files/Pods-project2/Pods-project2.debug.xcconfig
/Users/lovkumar/project/project2/ios/Pods/Target Support Files/Pods-project2-project2Tests/Pods-project2-project2Tests.release.xcconfig
/Users/lovkumar/project/project2/ios/Pods/Target Support Files/Pods-project2-project2Tests/Pods-project2-project2Tests.debug.xcconfig
Setting REACT_NATIVE build settings
Integrating client project
Pod installation complete! There are 59 dependencies from the Podfile and 62 total pods installed.
lovkumar@lovs-Mac-mini project2 % npx pod-install
Scanning for pods...
1.11.3
> pod install
Adding a custom script phase for Pod RNFBApp: [RNFB] Core Configuration
Auto-linking React Native modules for target `project2`: RNFBApp and RNFBAuth
[Codegen] Generating ./build/generated/ios/React-Codegen.podspec.json
Analyzing dependencies
[Codegen] Found FBReactNativeSpec
Downloading dependencies
Installing Firebase 9.1.0
Installing GoogleUtilities 7.7.0
Installing RNFBApp 15.0.0
Installing RNFBAuth 15.0.0
[!] The following Swift pods cannot yet be integrated as static libraries:

The Swift pod `FirebaseCoreInternal` depends upon `GoogleUtilities`, which does not define modules. To opt into those targets generating module maps (which is necessary to import them from Swift when building as static libraries), you may set `use_modular_headers!` globally in your Podfile, or specify `:modular_headers => true` for particular dependencies.
Couldn't install Pods. Updating the Pods project and trying again...
> pod install --repo-update
Adding a custom script phase for Pod RNFBApp: [RNFB] Core Configuration
Auto-linking React Native modules for target `project2`: RNFBApp and RNFBAuth
[Codegen] Generating ./build/generated/ios/React-Codegen.podspec.json
Updating local specs repositories
Analyzing dependencies
[Codegen] Found FBReactNativeSpec
Downloading dependencies
Installing Firebase 9.1.0
Installing GoogleUtilities 7.7.0
Installing RNFBApp 15.0.0
Installing RNFBAuth 15.0.0
[!] The following Swift pods cannot yet be integrated as static libraries:

The Swift pod `FirebaseCoreInternal` depends upon `GoogleUtilities`, which does not define modules. To opt into those targets generating module maps (which is necessary to import them from Swift when building as static libraries), you may set `use_modular_headers!` globally in your Podfile, or specify `:modular_headers => true` for particular dependencies.
Couldn't install Pods. Updating the Pods project and trying again...
Command `pod install` failed.
└─ Cause: The following Swift pods cannot yet be integrated as static libraries:

The Swift pod `FirebaseCoreInternal` depends upon `GoogleUtilities`, which does not define modules. To opt into those targets generating module maps (which is necessary to import them from Swift when building as static libraries), you may set `use_modular_headers!` globally in your Podfile, or specify `:modular_headers => true` for particular dependencies.

lovkumar@lovs-Mac-mini project2 % pod --version
1.11.3
lovkumar@lovs-Mac-mini project2 %

**Comment by mikehardy** [comment]
Hi there!

Unfortunately, if you look at this discussion entry with fresh eyes, it is incomprehensible, I can't help.

Please *search our issues* and read the documentation and consider opening an issue (not a discussion) filling out our template with this in mind https://stackoverflow.com/help/how-to-ask

You may find #6332 helpful if you are using Expo
